src/main/java/com/dynatrace/sdk/server/testautomation/models/TestRun.java

```diff
@@ -85,7 +85,10 @@
     @XmlAttribute
     private Integer numInvalidated;
 
-    @XmlElement(name = "testResults")
+    @XmlElement
+    private List<TestMetricFilter> includedMetrics = new ArrayList<>();
+
+    @XmlElement
```

Comment:
This is redundant after the change to api v2 (now field name is same as attribute name).
In api v1 the name was "testResult" to fit xml structure, I changed it when doing switch to v2, but did not notice now is same as field name.